fstat/static/js/app.js

```diff
@@ -0,0 +1,53 @@
+(function () {
+    $('input[name="daterange"]').daterangepicker({
+    locale: {format: "YYYY-MM-DD"},
+    }, function(start, end, label) {
+        start = start.format('YYYY-MM-DD');
+        end = end.format('YYYY-MM-DD');
+        var location = window.location.toString().split("?")[0];
+        window.location = location + `?start_date=${start}&end_date=${end}`;
+    });
+
+    function hasNaN(array) {
+        for (let i = 0; i < array.length; i++) {
+            if (isNaN(array[i])) {
+                return true;
+            }
+        }
+        return false;
+    }
+    listeners = {
+        showModal: function(fid, bugs) {
+            failure_id = fid
+            $('#associate-bug-modal').modal('show');
+            $('#bugIds').val(bugs);
+            $('#bugIds').tagsinput();
+        },
+        submit: function() {
+            if ($('#bugIds').val() !== "") {
+                var bugIds = $('#bugIds').val().split(',').map(function(bugId){return parseInt(bugId)});
+                    if (!hasNaN(bugIds)) {
+                        var URL = `/associate-bugs/${failure_id}`;
+                        $.ajax({
+                            url: URL,
+                            beforeSend: function(xhr){
+                                xhr.setRequestHeader("Content-Type","application/json");
+                            },
+                            data: JSON.stringify({bugIds: bugIds}),
+                            type: 'POST',
+                            success: function(data, status, xhr){
+                                location.reload();
+                            },
+                            error: function(err, status, xhr){
+                                alert(err.response);
+                            }
+                        });
+                    } else {
+                        alert("All BugIDs must be integer.")
```

Comment:
(This is fine to fix as a follow up) I'm not a huge fan of the alert boxes. Is there something that's less distracting we can do?